### DISCUSSION TITLE
v7 to v8: Do we need ProxyRenewalAgent ?

### DISCUSSION THREAD

**Discussion by marianne013** [DIRACGrid/DIRAC#7050] [Q&A] [answered]
Hi,

The upgrade instructions in the Wiki state:
_The new agent Framework/ProxyRenewalAgent has to be installed. The agent Framework/MyProxyRenewalAgent is instead discontinued and should be removed (its functionalities are part of ProxyRenewalAgent)._
We don't have MyProxyRenewalAgent installed, but from the phrasing it's not clear whether  ProxyRenewalAgent has to be installed regardless.

Cheers,
Daniela

**Comment by fstagni** [comment, marked as answer]
Sorry for the confusion. Yes, you need to install `ProxyRenewalAgent` nonetheless. I will amend now the wiki.

**Reply by marianne013** [reply]
Thanks. Looking at the code, it does tokens as well ?

**Reply by fstagni** [reply]
Those would be the "DIRAC tokens" which have nothing to do with the OAuth2 tokens (it is a functionality added long time ago that could probably be removed). It was only "just in case"

**Comment by marianne013** [comment]
While I have your attention:
The following services **can** be uninstalled:

    Framework/Monitoring
    Framework/Plotting

As far as I can tell they are gone in v8 and **have to be** uninstalled ?

**Reply by fstagni** [reply]
Correct. Fixing that too, thanks for the doc review.
